crates/aptos-faucet/cli/src/main.rs

```diff
@@ -74,7 +74,8 @@ impl FaucetCliArgs {
             Duration::from_secs(30),
             None,
             self.max_gas_amount,
-            30,
+            15,
```

Comment:
Should this be configurable in some way?

Reply:
This CLI doesn't get used I don't think so it's probably not necessary. It's configurable for the actual faucet which is what matters.